src/org/omegat/gui/exttrans/MachineTranslateTextArea.java

```diff
@@ -105,17 +122,58 @@ public MachineTranslateTextArea(IMainWindow mw) {
                 .collect(Collectors.toMap(GlossaryEntry::getSrcText, GlossaryEntry::getLocText));
     }
 
+    private void initDocument() {
+        StyleSheet baseStyleSheet = new StyleSheet();
+        HTMLEditorKit htmlEditorKit = new HTMLEditorKit();
+        baseStyleSheet.addStyleSheet(htmlEditorKit.getStyleSheet()); // Add default styles
+        Font font = getFont();
+        baseStyleSheet.addRule("body { font-family: " + font.getName() + "; "
+                + " font-size: " + font.getSize() + "; "
+                + " font-style: " + (font.getStyle() == Font.BOLD ? "bold"
+                        : font.getStyle() == Font.ITALIC ? "italic" : "normal") + "; "
```

Comment:
`font-style: bold` is not a thing; it's `font-weight: bold`.

Reply:
I see DictionariesTextArea currently uses `font-style: bold`. Maybe that's a special feature of Swing HTML?

Reply:
I double-checked with the following Groovy program and I was right: `font-style: bold` has no effect.

```groovy
import javax.swing.text.html.HTMLEditorKit;
import javax.swing.text.html.StyleSheet;
import javax.swing.JTextPane;
import javax.swing.JFrame;

def pane = new JTextPane();

def stylesheet = new StyleSheet();
def editorKit = new HTMLEditorKit();
stylesheet.addRule('body { font-style: bold }')
editorKit.setStyleSheet(stylesheet);
pane.setEditorKit(editorKit);
pane.setText('foo');

def frame = new JFrame();
frame.add(pane);
frame.pack();
frame.show();
```

Reply:
I've updated a stylesheet